sysvar: add 2 sysvars (#10662)
pingcap/docs: system-variables.md

- Scope: SESSION | GLOBAL
- Persists to cluster: Yes
- Default value: `utf8mb4_bin`

> - The default collation for a database when the database is created.

Reviewer: Need update according to the [latest Chinese](https://github.com/pingcap/docs-cn/pull/11505/files#diff-ec2cbb115cc58fe3afe2862eb5148732d323cbf7ae35dea97bd381a1f8d0e8a0R150)

Author: It is according to the latest Chinese: 该变量表示创建数据库时默认的排序规则。

Author: ```suggestion
- The default collation used when the database is created.
```

> ### identity

Reviewer: ```suggestion
### identity <span class="version-mark">New in v5.3.0</span>
```

- Note that if `SHARD_ROW_ID_BITS` and `PRE_SPLIT_REGIONS` have been set when a table is created, the specified number of Regions are evenly split after the table creation.

### tidb_shard_allocate_step <span class="version-mark">New in v5.0</span>

- Scope: SESSION | GLOBAL
- Persists to cluster: Yes
- Default value: `9223372036854775807`
- Range: `[1, 9223372036854775807]`
- This variable controls the maximum number of continuous IDs to be allocated for the [`AUTO_RANDOM`](/auto-random.md) or [`SHARD_ROW_ID_BITS`](/shard-row-id-bits.md) attribute. Generally, `AUTO_RANDOM` IDs or the `SHARD_ROW_ID_BITS` annotated row IDs are incremental and continuous in one transaction. You can use this variable to solve the hotspot issue in large transaction scenarios.

> ### tidb_simplified_metrics

Author: not in v6.2
